### ISSUE TITLE
Need help on fasttext.h atomic error 

### ISSUE THREAD

**Issue by prakritidev** [epfml/sent2vec#27]
Hi, 

I know this is silly problem but I just can't find how to solve this problem. 

`In file included from sent2vec.cpp:315:

./fasttext.h:18:10: fatal error: 'atomic' file not found
#include <atomic>
         ^~~~~~~~

1 error generated.
error: command '/usr/bin/clang' failed with exit status 1`

**Comment by prakritidev** [comment]
Yes, I figured it out after investing very long time. I forgot to close this issue. 

My apologies.

**Comment by ahmdtaha** [comment]
I faced the same problem, Adding [this flag](https://stackoverflow.com/questions/36960587/cython-build-cant-find-c11-stl-files-but-only-when-called-from-setup-py) solved it for me
